sdk/translation/azure-ai-translation-text/tests/test_translation.py

```diff
@@ -319,3 +319,21 @@ def test_translate_aad(self, **kwargs):
         assert len(response[0].translations) == 1
         assert response[0].translations[0].to == "cs"
         assert response[0].translations[0].text is not None
+
+    @pytest.mark.skip
```

Comment:
Why are we skipping this test?

Reply:
We have internal work item for the future to fix the test resource creation that will enable us to remove the skip.